dags/snowflake_examples/taxi_snowflake_el.py

```diff
@@ -0,0 +1,294 @@
+from airflow import DAG
+from airflow.decorators import task
+from airflow.models import Variable
+from airflow.models.baseoperator import chain
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.utils.dates import datetime
+from airflow.utils.task_group import TaskGroup
+from airflow.providers.amazon.aws.transfers.local_to_s3 import LocalFilesystemToS3Operator
+from airflow.providers.snowflake.operators.snowflake import SnowflakeOperator
+from airflow.providers.snowflake.transfers.s3_to_snowflake import S3ToSnowflakeOperator
+from plugins.snowflake_check_operators import (
```

Comment:
why's this coming from a plugin?

Reply:
I think I just need to upgrade my current Airflow to get it to recognize the `SnowflakeCheckOperators` I PR'd in OSS, but even if it does, it won't recognize all the ones I need, as there are a couple that don't exist as Snowflake versions (that PR got closed in favor of the SQL Operator change I made). That SQL Operator change _also_ isn't released yet, so this plugin gives all the functionality for now, and I will make caveats about that in the webinar that this is a hack while we wait for the mature functionality to get released. (Hopefully that PR is closed before the webinar... I been pinging committers).